crt_portal/static/sass/custom/alerts.scss

```diff
@@ -2,13 +2,14 @@
 
 .page-errors {
   font-weight: bold;
-  color: $red-vivid-60;
+  color: color($theme-color-secondary-dark);
```

Comment:
I was a little torn on how to do this. On one hand, using the USWDS `color()` function in the CSS rules themselves isn't going to be intuitive to new front-end devs. But on the other, the alternative is to use the `color()` function in the child theme colors file (i.e., `$theme-color-primary: color('blue-warm-70v')`), which then means you can't use the color variable in USWDS mixins (like `@u-bg($theme-color-primary)`).

Reply:
I think what you have is the best compromise. Agree doing `color()` is not the most intuitive but it's definitely good enough to move forward with.